src/app/Table.elm

```diff
@@ -1,29 +1,91 @@
-module Table exposing (view, init, Msg)
+module Table exposing (view, init, initWithPlayersAndCup)
+
+{-|
+
+   This module handles the table of players,
+   and everything relating to that.
+
+-}
 
 import Html exposing (..)
+import Html.Events exposing (..)
+import Types exposing (PlayerID)
+import TableMsg exposing (Msg(..))
+import Dice
 
 
-{-
+type alias PlayerName =
+    String
 
-   This module handles the view rendering for the table of players
-   and everything relating to that.
 
--}
+type alias DiceCount =
+    Int
+
 
+type alias TablePlayer =
```

Comment:
I wanted to make it explicitly clear that this is *not* a model, it's a view model. That is, it's the data for the visual representation of a player, not the data we store about a player from the server, or any other kind of representation we might have of a player. Perhaps it should be called ViewPlayer, but I find that name really unclear.

So, later on, we might add `position` to this `TablePlayer`. From that perspective, it should be pretty clear that that data is nothing to do with the player themselves.